docker_build/api/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.10-slim AS base
+
+ARG DEBIAN_FRONTEND=noninteractive
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libgl1 libpq-dev ffmpeg wget mesa-common-dev libegl1-mesa-dev libgles2-mesa-dev mesa-utils \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-cache-dir mediapipe==0.10.14 requests opencv-python-headless
+
+RUN mkdir /worker_models \
+    && wget https://storage.googleapis.com/mediapipe-models/pose_landmarker/pose_landmarker_heavy/float16/latest/pose_landmarker_heavy.task \
+        -O /worker_models/pose_landmarker_heavy.task \
+    && wget https://storage.googleapis.com/mediapipe-models/face_landmarker/face_landmarker/float16/latest/face_landmarker.task \
+        -O /worker_models/face_landmarker.task \
+    && wget https://storage.googleapis.com/mediapipe-models/hand_landmarker/hand_landmarker/float16/latest/hand_landmarker.task \
+        -O /worker_models/hand_landmarker.task
+
+RUN pip install --no-cache-dir ultralytics==8.3.78 --no-deps
+RUN pip install --no-cache-dir psutil matplotlib pandas tqdm pyyaml
+RUN pip install --no-cache-dir torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+RUN pip install --no-cache-dir fastapi uvicorn
+
+RUN wget https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11x-pose.pt -O /worker_models/yolo11x-pose.pt
+
+RUN pip install --no-cache-dir fastapi uvicorn[standard] python-multipart 
+
+COPY ./worker /app
+
+WORKDIR /app
+
+CMD ["uvicorn", "api:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
Just one tiny note here, since those docker files are very similar it would be beneficial to not replicate them but use multi-stage build or some kind of base-child-image setup to build both api and cli and potentially also worker from the same base file. But this is just a sidenote, no need to bother with this now while you want to make progress